core-api/src/test/resources/definitions/exposed-services.json

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
I think you should test from the main/resources because these files can be different.